README.md

```diff
@@ -71,15 +71,15 @@ For detailed description of the requirements and the scripts along with examples
 # Using ONBUILD images
 
 In case you want to use one of the official Docker language stack images for
-your build you don't have do anything extra. The STI is capable to recognize the
-Docker image with ONBUILD instructions and choose the OnBuild strategy. This
+your build you don't have do anything extra. STI is capable of recognizing the
+Docker image with ONBUILD instructions and choosing the OnBuild strategy. This
 strategy will trigger all ONBUILD instructions and execute the assemble script
 (if it exists) as the last instruction.
 
 Since the ONBUILD images usually don't provide any entrypoint, in order to use
-this build strategy you have to provide it. You can either include the 'run',
+this build strategy you will have to provide one. You can either include the 'run',
 'start' or 'execute' script in your application source root folder or you can
-specify a valid STI scripts URL and the 'run' script will be fetched and set as
+specify a valid STI script URL and the 'run' script will be fetched and set as
```

Comment:
eh, again debatable.  it's a single script url.  what is located at that url can be one or more scripts.

just like a "source url" contains sources but we don't call it a sources url.